### PR TITLE
Feat/Tabs component

### PR DESCRIPTION
**Pull request by pritam248**
### Screenshots
Screen Shot 2020-09-30 at 2 50 53 PM
Screen Shot 2020-09-30 at 2 50 44 PM

## Checklist

- [x] PR is up-to-date with a description of changes and screenshots (if applicable).
- [x] All files are lint-free.
- [x] Followed Semantic PR guidelines.
- [x] Tested locally for regressions.

### CODE REVIEW

File: docs/docs/tabs.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+id: tabs
+title: Tabs
+sidebar_label: Tabs
+---
+
+##### Utilities for Buttons
```
Comment (anshul-pinto0410) [inline docs/docs/tabs.md]: utilities for tabs

**Review by anshul-pinto0410 — COMMENTED** [review]
the grey bottom border for non active tabs is not there

**Comment by pritam248** [comment]
> the grey bottom border for non active tabs is not there

oh seems to be border in screenshot but its not .....its a Scrollbar :-D

File: docs/docs/tabs.md (hunk 1 above)
Comment (pritam248) [reply docs/docs/tabs.md]: oops...will fix

**Comment by pritam248** [comment]
@anshul-pinto0410 can we merge this now ?

**Review by anshul-pinto0410 — APPROVED** [review]

**Pull request merged**